api/index.js

```diff
@@ -12,56 +12,46 @@ var errors = require('./lib/errors');
 var serverLib = require('./lib/server-lib');
 var remote = require('./lib/remote');
 
-module.exports = {
-  info: {
-    serverStatus: Info.serverStatus,
-    isConnected: Info.isConnected,
-    fee: Info.fee,
-    uuid: Info.uuid
-  },
+function RippleAPI() {
+  this.remote = remote;
+}
```

Comment:
This pattern of using `this` as a global scope in all the modules in `api/*` is confusing.

It might be nicer to do something like:

```
var remote = require('remote')
var Info = require('./info')(remote);
```

And info looks like:

```
module.exports = function(client) {
...
}
```

Then it's clear where the `client` is coming from. I took this pattern from the Twilio rest API client: https://github.com/twilio/twilio-node/blob/master/lib/resources/Accounts.js